$NetBSD: patch-configure,v 1.1 2013/02/06 18:07:04 jperkin Exp $

* Fix sys/bufmod.h and linking issues on Solaris.
* Use standard manpage layout on all platforms.

--- configure.orig	2012-03-29 12:57:32.000000000 +0000
+++ configure
@@ -5225,6 +5225,20 @@ dlpi)
 	# Also, due to the bug above applications that link to libpcap with
 	# libdlpi will have to add "-L/lib" option to "configure".
 	#
+			for ac_header in sys/bufmod.h sys/dlpi_ext.h
+do :
+  as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
+ac_fn_c_check_header_mongrel "$LINENO" "$ac_header" "$as_ac_Header" "$ac_includes_default"
+if eval test \"x\$"$as_ac_Header"\" = x"yes"; then :
+  cat >>confdefs.h <<_ACEOF
+#define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
+_ACEOF
+
+fi
+
+done
+		 	SSRC="dlpisubs.c"
+
 	saved_ldflags=$LDFLAGS
 	LDFLAGS="$LIBS -L/lib"
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for dlpi_walk in -ldlpi" >&5
@@ -5763,19 +5777,6 @@ else
 		case "$V_PCAP" in
 
 		dlpi|libdlpi)
-			for ac_header in sys/bufmod.h sys/dlpi_ext.h
-do :
-  as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
-ac_fn_c_check_header_mongrel "$LINENO" "$ac_header" "$as_ac_Header" "$ac_includes_default"
-if eval test \"x\$"$as_ac_Header"\" = x"yes"; then :
-  cat >>confdefs.h <<_ACEOF
-#define `$as_echo "HAVE_$ac_header" | $as_tr_cpp` 1
-_ACEOF
-
-fi
-
-done
-
 			#
 			# This might be Solaris 8 or later, with
 			# SIOCGLIFCONF, or it might be some other OS
@@ -5820,7 +5821,6 @@ $as_echo "$ac_cv_lbl_have_siocglifconf"
 			#
 			# Needed for common functions used by pcap-[dlpi,libdlpi].c
 			#
-		 	SSRC="dlpisubs.c"
 			;;
 
 		*)
@@ -6688,8 +6688,6 @@ fi
 # Individual cases can override this.
 #
 DYEXT="none"
-MAN_FILE_FORMATS=5
-MAN_MISC_INFO=7
 case "$host_os" in
 
 aix*)
@@ -6916,6 +6914,9 @@ $as_echo "#define HAVE_SOLARIS 1" >>conf
 	;;
 esac
 
+MAN_FILE_FORMATS=5
+MAN_MISC_INFO=7
+
 if test -n "$ac_tool_prefix"; then
   # Extract the first word of "${ac_tool_prefix}ranlib", so it can be a program name with args.
 set dummy ${ac_tool_prefix}ranlib; ac_word=$2
